vuepress/docs/next/docs/component-generator/blueprint-features.md

```diff
@@ -0,0 +1,135 @@
+# Entando Blueprint Features
+
+The Entando Blueprint makes it easier and faster to customize your application by   generating controllers, repositories, services, and micro frontends for your entity. It means generating those project files in minutes simply by answering questions that define the parameters of your applications. 
```

Comment:
extra space?
entity sounds odd... i think you can stop the sentence at "frontends"; application is implied
add comma: "in minutes, simply"
applications -> application (singular to be consistent)